include/erlavro.hrl

```diff
@@ -48,7 +48,11 @@
          N =:= ?AVRO_DOUBLE  orelse
          N =:= ?AVRO_BYTES)).
 
+-ifdef(_ERLAVRO_ELIXIR_).
```

Comment:
This identifier isn't defined anywhere in this repo, so this doesn't make sense.
Do you want to change the expansion of `?AVRO_REQUIRED` based on external preferences? If so then you should do
```
-ifndef(AVRO_REQUIRED).
-define(AVRO_REQUIRED, <existing default>).
-endif.
```

Reply:
Good point. That's probably the better approach to overwrite this from within elixir.